extension/src/setup/index.ts

```diff
@@ -546,21 +546,17 @@ export class Setup
   }
 
   private watchDotFolderForChanges() {
-    const cwd = getFirstWorkspaceFolder()
-
-    if (!cwd) {
-      return
-    }
-
     const disposer = Disposable.fn()
     this.dotFolderWatcher = disposer
     this.dispose.track(this.dotFolderWatcher)
 
-    return createFileSystemWatcher(
-      disposable => disposer.track(disposable),
-      getRelativePattern(cwd, '**'),
-      path => this.dotFolderListener(disposer, path)
-    )
+    for (const workspaceFolder of getWorkspaceFolders()) {
```

Comment:
[F] There was a small chance of a bug here when the user has multiple workspace folders. We can easily watch multiple folders for changes.